hazelcast-jet-core/src/main/java/com/hazelcast/jet/processor/Sources.java

```diff
@@ -74,6 +101,22 @@ public static ProcessorMetaSupplier readMap(@Nonnull String mapName, @Nonnull Cl
     }
 
     /**
+     * Returns a meta-supplier of processor that will fetch entries from a
+     * Hazelcast {@code IMap} in a remote cluster. Processor will emit the
+     * entries as {@code Map.Entry}.
+     * <p>
+     * If the underlying map is concurrently modified, there may be missing or
+     * duplicate items.
+     */
+    @Nonnull
+    public static <K, V, T> ProcessorMetaSupplier readMap(@Nonnull String mapName,
```

Comment:
We can also mention the full note from partition iterator: 
>Iterating the map should be done only when the {@link IMap} is not being
mutated and the *cluster is stable* (there are no migrations or membership changes).
In other cases, the iterator may not return some entries or may return an entry twice.